client/src/services/helper.js

```diff
@@ -0,0 +1,26 @@
+
+
+export const BASE_URL = ''; // or url for staging
+
+export const makeRequest = (path, options) => {
+
+  if(!options) {
+    options = {};
+  }
+
+  options.accept = 'application/json';
+
+  const token = localStorage.getItem('token');
+
+  if (token) {
+    if (!options.headers) {
+      options.headers = {};
+    }
+    options.headers['Authorization'] =  `Bearer ${ token }`;
+  }
+
+  return fetch(BASE_URL + path, options).then(response => {
```

Comment:
what do you think about using axios ?

Reply:
What advantage does axios offer over fetch. Fetch is native so it doesnt need to shipped with the sourcecode thats def an advantage

Reply:
1) axios performs automatic transforms of JSON data.
2) in axios we can cancel requests.
?

Reply:
Not completely sold as yet. Fetch supports cancellation. And .json() transforms json data as well. Something to keep on the list, but at the moment I think fetch should do.